src/lib/components/chat/Settings/Models.svelte

```diff
@@ -599,12 +613,25 @@
 								<div class="flex flex-col">
 									<div class="font-medium mb-1">{model}</div>
 									<div class="">
+										<div class="flex flex-row space-x-4">
 										<div
 											class="dark:bg-gray-600 bg-gray-500 text-xs font-medium text-gray-100 text-center p-0.5 leading-none rounded-full"
 											style="width: {Math.max(15, modelDownloadStatus[model].pullProgress ?? 0)}%"
 										>
 											{modelDownloadStatus[model].pullProgress ?? 0}%
 										</div>
+										<button
+										class="px-2.5 bg-gray-100 hover:bg-gray-200 text-gray-800 dark:bg-gray-850 dark:hover:bg-gray-800 dark:text-gray-100 rounded-lg transition"
```

Comment:
Q: If the UI of **X** needs to be changed, please tell me the class as I am still trying to figure out.

Reply:
Updated!